tools/theme/src/express/core.ts

```diff
@@ -14,4 +14,7 @@ import { Theme } from '../Theme.js';
 
 import coreStyles from './theme.css.js';
 
+Theme.registerThemeFragment('express', 'system', coreStyles);
```

Comment:
I _think_ we want only the newer instance?

Reply:
_Nevermind_. This was something I was testing, it accidentally got into the PR